blobstore-nats/src/lib.rs

```diff
@@ -0,0 +1,415 @@
+//! HTTP Blobstore Example
```

Comment:
Having an example of interacting with a blobstore programmatically can be helpful, but for our blogpost, I'm hoping we can use Blobby to show a more real-world example that's dynamic. First plain curl but then later perhaps via a simple UI.